tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -378,31 +365,17 @@ class InternalCallMapHandlerTest extends TestCase
         'pdo::sqlitecreateaggregate',
         'pdo::sqlitecreatecollation',
         'pdo::sqlitecreatefunction',
-        'pdostatement::__sleep',
-        'pdostatement::__wakeup',
         'simplexmlelement::__get',
         'simplexmlelement::offsetexists',
         'simplexmlelement::offsetget',
         'simplexmlelement::offsetset',
         'simplexmlelement::offsetunset',
         'spldoublylinkedlist::__construct',
-        'splfileinfo::__wakeup',
-        'splfixedarray::current',
-        'splfixedarray::key',
-        'splfixedarray::next',
-        'splfixedarray::rewind',
-        'splfixedarray::valid',
         'splheap::__construct',
         'splmaxheap::__construct',
         'splobjectstorage::__construct',
         'splpriorityqueue::__construct',
         'splstack::__construct',
-        'weakmap::__construct',
-        'weakmap::current',
-        'weakmap::key',
-        'weakmap::next',
-        'weakmap::rewind',
-        'weakmap::valid',
```

Comment:
No idea where these came from. WeakMap was added in PHP 8. I added the correct methods.